sp_HumanEvents/sp_HumanEventsBlockViewer.sql

```diff
@@ -640,8 +710,25 @@ SELECT
             RTRIM(kheb.object_id)
         ),
     kheb.activity,
-    kheb.spid,
-    kheb.ecid,
+    blocking_tree = 
+        REPLICATE (' > ', kheb.blocking_level) + 
+        CASE kheb.activity
+            WHEN 'blocking' 
+            THEN '(' + kheb.blocking_desc + ') is blocking (' + kheb.blocked_desc + ')'
+            ELSE ' > (' + kheb.blocked_desc + ') is blocked by (' + kheb.blocking_desc + ')'
+        END,
+    spid = 
```

Comment:
`spid` and `ecid` now take the value of either `blocked_spid` and `blocked_ecid` or `blocking_spid` and `blocking_ecid`. Whichever is appropriate.